add document to run test using minikube for lvmd daemonset (#318)
topolvm/topolvm: e2e/README.md

=====================================

This directory contains codes for end-to-end tests of TopoLVM.

> The test run using [kind (Kubernetes IN Docker)][kind] to make an environment with multiple `lvmd` running as systemd service.

Reviewer: ```suggestion
The tests run using [kind (Kubernetes IN Docker)][kind] to make an environment with multiple `lvmd` running as systemd service.
```

> In another hand, to test `lvmd` running as a daemonset, [minikube][minikube] is used to make a test environment in the localhost without container or VM.

Reviewer: ```suggestion
On the other hand, to test `lvmd` running as a daemonset, [minikube][minikube] is used to make a test environment in the localhost without container or VM.
```

How to run tests
----------------

> ### Run tests using kind with lvmd as systemd service

Reviewer: ```suggestion
### Run tests using kind with lvmd as a systemd service
```

Start `lvmd` as a systemd service as follows:

```console
make start-lvmd
```

> Run the test with the following command. Repeat it until you get satisfied.

Reviewer: ```suggestion
Run the tests with the following command. Repeat it until you get satisfied.
```

When tests fail, use `kubectl` to inspect the Kubernetes cluster.

```console
make test
```

You can cleanup test environment as follows:

```
# stop Kubernetes
make shutdown-kind

# stop lvmd
make stop-lvmd
```

> ### Run tests using minikube with lvmd as daemonset

Reviewer: ```suggestion
### Run tests using minikube with lvmd as a daemonset
```

Make lvm and launch Kubernetes using minikube with the following commands:

```console
make daemonset-lvmd/create-vg
make daemonset-lvmd/setup-minikube
make daemonset-lvmd/update-minikube-setting
```

> Run the test with the following command.

Reviewer: ```suggestion
Run the tests with the following command.
```
